2021-01-16 15:28:39,233 INFO    MainThread:2175371 [wandb_setup.py:_flush():70] setting env: {}
2021-01-16 15:28:39,233 INFO    MainThread:2175371 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-16 15:28:39,233 INFO    MainThread:2175371 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-16 15:28:39,234 INFO    MainThread:2175371 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/FedAvgExp/wandb/run-20210116_152839-1awws2na/logs/debug.log
2021-01-16 15:28:39,234 INFO    MainThread:2175371 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/FedAvgExp/wandb/run-20210116_152839-1awws2na/logs/debug-internal.log
2021-01-16 15:28:40,899 INFO    MainThread:2175371 [wandb_run.py:_console_start():1308] atexit reg
2021-01-16 15:28:40,899 INFO    MainThread:2175371 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [redirect.py:install():205] install start
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [redirect.py:install():220] install stop
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [redirect.py:install():205] install start
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [redirect.py:install():220] install stop
2021-01-16 15:28:40,900 INFO    MainThread:2175371 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-16 15:28:40,901 INFO    MainThread:2175371 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-16 15:28:40,901 INFO    MainThread:2175371 [wandb_run.py:_restore():1250] restore
2021-01-16 15:28:40,901 INFO    MainThread:2175371 [redirect.py:uninstall():224] uninstall start
2021-01-16 15:28:40,901 INFO    MainThread:2175371 [redirect.py:_stop():279] _stop: stdout
2021-01-16 15:28:40,901 INFO    stdout    :2175371 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-16 15:28:40,901 INFO    MainThread:2175371 [redirect.py:_stop():285] _stop closed: stdout
2021-01-16 15:28:40,902 INFO    stdout    :2175371 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():291] _stop joined: stdout
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:uninstall():228] uninstall done
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:uninstall():224] uninstall start
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():279] _stop: stderr
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():285] _stop closed: stderr
2021-01-16 15:28:40,902 INFO    stderr    :2175371 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-16 15:28:40,902 INFO    stderr    :2175371 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():291] _stop joined: stderr
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-16 15:28:40,902 INFO    MainThread:2175371 [redirect.py:uninstall():228] uninstall done
2021-01-16 15:28:41,652 INFO    MainThread:2175371 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "ef82c28925384327a4a0fcaa0f133c84"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1102
      total_bytes: 1102
    }
  }
}

2021-01-16 15:28:43,656 INFO    MainThread:2175371 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "3a724f44ebee42da81b651825632ae5c"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 3839
      total_bytes: 3839
    }
  }
}

2021-01-16 15:28:44,703 INFO    MainThread:2175371 [wandb_run.py:_show_files():1614] logging synced files
2021-01-16 15:28:44,704 INFO    MainThread:2175371 [internal.py:handle_exit():139] Internal process exited
